src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Hex2Dec.txt

```diff
@@ -0,0 +1,57 @@
+﻿>> Hex2Dec("0")
+0
+
+>> Hex2Dec(Blank())
+#Error(Kind=InvalidArgument)
+
+// Positive Integer Parameters
```

Comment:
This should be 0 - that's what Excel does. We should also have a test with `Hex2Dec("")` which should have the same result.